model-optimizer/extensions/front/tf/TFSliceToSlice.py

```diff
@@ -43,8 +44,12 @@ def replace_sub_graph(self, graph: Graph, match: dict):
         rename_nodes([(node, slice_name + '/to_be_removed'), (slice_node, slice_name)])
 
         eq_node = Equal(graph, {'name': slice_name + '/equal'}).create_node()
-        minus_one_node = Const(graph, {'name': slice_name + '/minus_one', 'value': np.array(-1)}).create_node()
-        int32_max_node = Const(graph, {'name': slice_name + '/int32_max', 'value': np.iinfo(np.int32).max}).create_node()
+        minus_one_node = Const(graph, {'name': slice_name + '/minus_one', 'value': int64_array(-1)}).create_node()     
+        
+        #Select requires equal dtypes on ports 1 and 2.
+        #Dtype of [2] port is int64(summ_node -> cast to int64), so dtype of [1] (int32_max_node)  must be the same
```

Comment:
I suggest to remove this line and leave only comment on line 49. Other changes are ok. Approved in advance.